src/simulator_core/simulation/plotting.py

```diff
@@ -0,0 +1,98 @@
+#  Copyright (c) 2023. Deltares & TNO
+#
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+"""Plot result on the map for debugging."""
+import esdl
+import plotly.express as px
+import geopandas as gpd
+import numpy as np
+from shapely.geometry import LineString
+
+
+class Plotting:
+    """Plotting plot the results of asset (currently pipe) for inspection."""
+
+    @staticmethod
+    def plot_map(assets: list):
```

Comment:
I believe you should put assets: List[AbstractAssets] from the asset_abstract.py

Reply:
done